code/code/misc/skill_dam.cc

```diff
@@ -150,7 +150,7 @@ if (discArray[skill]->disc == discArray[skill]->assDisc) {
 
   // cut area effects in half
   if (IS_SET(discArray[skill]->targets, TAR_AREA)) 
-    fixed_amt *= 0.75;
+    fixed_amt /= 2.0;
```

Comment:
I just changed this. Why would you change it back?

Reply:
I didn't intentionally - my guess is that this code just didn't get that update.